earth_enterprise/src/installer/install_server.sh

```diff
@@ -109,10 +218,322 @@ main_postinstall()
      python ./set_geecheck_config.py
   fi
 
-  # 10)
+  # 10) 
   show_final_success_message
 }
 
+#-----------------------------------------------------------------
+# Pre-install Functions
+#-----------------------------------------------------------------
+
+show_intro()
+{	
+	echo -e "\nWelcome to the $GEES $LONG_VERSION installer."
+}
+
+# TODO: Update for GEE Server
+show_help()
+{
+	echo -e "\nUsage: \tsudo ./install_fusion.sh [-dir /tmp/fusion_os_install -ar /gevol/assets -sv /gevol/src -u fusionuser"
+	echo -e "\t\t-g gegroup -nobk -nop -hnf -nostart]\n"
+
+	echo -e "-h \t\tHelp - display this help screen"
+	echo -e "-dir \t\tTemp Install Directory - specify the temporary install directory. Default is [$TMPINSTALLDIR]."	
+	echo -e "-u \t\tFusion User Name - the user name to use for Fusion. Default is [$GEFUSIONUSER_NAME]. \n\t\tNote: this is only used for new installations."
+	echo -e "-g \t\tUser Group Name - the group name to use for the Fusion user. Default is [$GROUPNAME]. \n\t\tNote: this is only used for new installations."
+	echo -e "-ar \t\tAsset Root Mame - the name of the asset root volume.  Default is [$ASSET_ROOT]. \n\t\tNote: this is only used for new installations. Specify absolute paths only."
+	echo -e "-sv \t\tSource Volume Name - the name of the source volume.  Default is [$SOURCE_VOLUME]. \n\t\tNote: this is only used for new installations. Specify absolute paths only."
+	echo -e "-nobk \t\tNo Backup - do not backup the current fusion setup. Default is to backup \n\t\tthe setup before installing."
+	echo -e "-nop \t\tNo Purge - do not delete the temporary install directory upon successful run of the installer."
+	echo -e "\t\tDefault is to delete the directory after successful installation."
```

Comment:
This is still useful and should probably remain.  We don't want to assume that every server will have both server and fusion.  This gives the user an option to not remove the temporary install directory (default is to remove it).